src/Pos/Generator/Block/Mode.hs

```diff
@@ -167,6 +180,18 @@ mkBlockGenContext bgcParams@BlockGenParams{..} = do
         bgcTxpMem <- (,ignoreTxpMetrics) <$> mkTxpLocalData
         bgcDelegation <- mkDelegationVar @SscGodTossing
         return BlockGenContext {..}
+  where
+    -- Genesis utxo is needed only for boot era stakeholders
+    bgcGenesisUtxo =
```

Comment:
See the comment. The `GenesisUtxo` used in `TestParams` shouldn't have the "genesis" prefix. It is a utxo snapshot corresponding to the beginning of block generation. 
`bgcGenesisUtxo` is _really genesis_. But it's used only for boot era stakeholders as it is written in the comment above.